receiver/flynn-receive.go

```diff
@@ -103,11 +104,15 @@ func main() {
 		proc := prevRelease.Processes[t]
 		proc.Cmd = []string{"start", t}
 		if t == "web" {
-			proc.Ports = []ct.Port{{Port: 8080, Proto: "tcp"}}
-			if proc.Env == nil {
-				proc.Env = make(map[string]string)
-			}
-			proc.Env["SD_NAME"] = app.Name + "-web"
+			proc.Ports = []ct.Port{{
+				Port:  8080,
+				Proto: "tcp",
+				Service: &host.PortService{
+					Name:   app.Name + "-web",
+					Create: true,
+					Check:  &host.PortServiceCheck{Type: "http"},
```

Comment:
Yeah, probably easier to just do TCP and won't show up in the logs, we can always change it later.